helper/Cargo.toml

```diff
@@ -0,0 +1,41 @@
+[package]
+name = "helper"
+version = "0.1.0"
+rust-version = "1.64.0"
+edition = "2021"
+
+[features]
+default = ["web-app", "cli"]
+web-app = ["hyper", "hyper-tls", "sec1", "time"]
+cli = ["clap", "comfy-table"]
+test-fixture = []
+
+[dependencies]
+ipa = { version = "*", path = "../ipa", features = ["real-world-infra"] }
+clap = { version = "4.0", optional = true, features = ["derive"] }
+comfy-table = { version = "7.0", optional = true }
+hyper = { version = "0.14.26", optional = true, features = ["client", "h2", "stream"] }
+hyper-tls = { version = "0.5.0", optional = true }
+# TODO(640): sec1 only needed until we standardize on rustls
+sec1 = { version = "0.7", features = ["alloc", "pem", "pkcs8"], optional = true }
+time = { version = "0.3", optional = true }
+tokio = { version = "1.28", features = ["rt", "rt-multi-thread", "macros"] }
+tracing = "0.1.37"
+generic-array = "0.14.7"
```

Comment:
we could specify common dependencies inside the workspace toml file: https://doc.rust-lang.org/nightly/cargo/reference/specifying-dependencies.html#inheriting-a-dependency-from-a-workspace

Reply:
`ipa` and `helper` have many in common, but `ipa-macros` only depend on syn and quote.